danuluma/dannstore
# #161247582 add usage examples to readme

#### What does this PR do?
Add usage and testing instructions

#### Description of Task to be completed?
Add examples of the endpoints usage and testing

#### How should this be manually tested?
* Open your favourite web browser
* Navigate to this repo https://github.com/danuluma/dannstore
* Scroll to the bottom
* You should see the readme on the lower part of the repo

#### What are the relevant pivotal tracker stories?
[#161247582](https://www.pivotaltracker.com/story/show/161247582)
